langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModelIT.java

```diff
@@ -356,6 +370,10 @@ void should_handle_parallel_function_calls() {
             .project(System.getenv("GCP_PROJECT_ID"))
             .location(System.getenv("GCP_LOCATION"))
             .modelName(GEMINI_1_5_PRO)
+            .temperature(0f)
```

Comment:
nit: I would change the type of `temperature` to `Double` (and all other `Float` parameters) to avoid this nasty `f` at the end

Reply:
It's been that way since the first version, but I think it's not a breaking change, so ok.

Reply:
This applies to `topP` as well then.

Reply:
But internally, Gemini uses floats for those.

Reply:
Ah ok then

Reply:
I made the change but then reverted back, to avoid any breaking change.